### PR TITLE
Paciak nocny skin

### PR DESCRIPTION
**Pull request by ic3fresh**
fix "wyloguj się"
fix cookie popup colors
fix calendar today colors

### CODE REVIEW

File: public/skin-paciak-nocny.css [hunk 1]
```diff
@@ -6767,8 +6769,8 @@ button.close {
   text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.3);
 }
 .navbar .navbar-nav > li > a {
-  border-right: 1px solid rgba(0, 0, 0, 0.2);
-  border-left: 1px solid rgba(255, 255, 255, 0.1);
+  border-right: 0px solid rgba(0, 0, 0, 0.2);
+  border-left: 0px solid rgba(255, 255, 255, 0.1);
```
Comment (mateusz-lisik) [inline public/skin-paciak-nocny.css]: Zero values shouldn't have unit provided

**Review by mateusz-lisik — COMMENTED** [review]
Everything but those 0px units seems to be ok. I guess importants are required to override default theme values.

**Comment by ic3fresh** [comment]
Deleted unnecessary lines.

**Review by mateusz-lisik — APPROVED** [review]
Looks ok now!

**Pull request merged**